web/vtadmin/src/components/routes/tablet/Advanced.tsx

```diff
@@ -109,163 +108,123 @@ const Advanced: React.FC<AdvancedProps> = ({ alias, clusterID, tablet }) => {
-                                href="https://vitess.io/docs/reference/programs/vtctl/tablets/#stopreplication"
-                                target="_blank"
-                                rel="noreferrer"
-                                className="text-gray-900"
-                            >
-                                <span className="text-sm font-semibold text-gray-900">Documentation</span>{' '}
-                                <Icon
-                                    icon={Icons.open}
-                                    className="ml-1 inline-block h-6 w-6 text-gray-900 fill-current"
-                                />
-                            </a>
-                        </div>
-                        <p className="text-base m-0">
-                            This will run the underlying database command to stop replication on tablet{' '}
-                            <span className="font-bold">{alias}</span>. For example, in mysql 8, this will be{' '}
-                            <span className="font-mono text-sm p-1 bg-gray-100">stop replication</span>.
-                        </p>
-                        {primary && (
-                            <p className="text-danger">
-                                <Icon icon={Icons.alertFail} className="fill-current text-danger inline mr-2" />
-                                Command StopTablet cannot be run on the primary tablet.
-                            </p>
-                        )}
-                        <button
-                            onClick={() => stopReplicationMutation.mutate()}
-                            className="btn btn-secondary mt-4"
-                            disabled={primary || stopReplicationMutation.isLoading}
-                        >
-                            Stop replication
-                        </button>
-                    </div>
+                <div>
+                    <ActionPanel
+                        confirmationValue=""
+                        description={
+                            <>
+                                This will run the underlying database command to start replication on tablet{' '}
+                                <span className="font-bold">{alias}</span>. For example, in mysql 8, this will be{' '}
+                                <span className="font-mono text-sm p-1 bg-gray-100">start replication</span>.
+                            </>
+                        }
+                        disabled={primary}
+                        documentationLink="https://vitess.io/docs/reference/programs/vtctl/tablets/#startreplication"
+                        loadedText="Start Replication"
+                        loadingText="Starting Replication..."
+                        mutation={startReplicationMutation as UseMutationResult}
+                        title="Start Replication"
+                        warnings={[primary && 'Command StartReplication cannot be run on the primary tablet.']}
```

Comment:
I noticed that this copy is currently:

> Command StartTablet cannot be run on the primary tablet.

I _think_ it should probably be (emphasis mine)

> Command **StartReplication** cannot be run on the primary tablet.

... But wanted to call this out in case I'm missing something. Same deal with the `StopTablet` -> `StopReplication` copy change below.